services/harmony/test/util/edl-api.ts

```diff
@@ -0,0 +1,34 @@
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+import {
+  getEdlGroupInformation,
+} from '../../app/util/edl-api';
+import logger from '../../app/util/log';
+import { stubEdlRequest, token, unstubEdlRequest } from '../helpers/auth';
+
+describe('util/edl-api', function () {
+  describe('getEdlGroupInformation', function () {
+    before(function () {
+      stubEdlRequest(
+        '/oauth/token',
+        { grant_type: 'client_credentials' },
+        token({ accessToken: 'fake_access' }),
+      );
+    });
+    after(function () {
+      unstubEdlRequest();
+    });
+    describe('when the user is not part of the service deployers group', function () {
```

Comment:
I'd sanity check a few tests:
1.) only in service-deployer group
2.) in service-deployer and another group
3.) not in any group

You have 2 and 3 covered it looks like, so I'd probably add one more.